exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/MemoryUtils.java

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
Please do not use javadoc comment.

Reply:
Vlad, can you please explain what you meant? are you only referring for specific members or is it a general comment?

Reply:
I refer to `/**`, please use `/*`